diesel/src/pg/types/date_and_time/time.rs

```diff
@@ -44,6 +49,23 @@ impl ToSql<Timestamp, Pg> for PrimitiveDateTime {
     }
 }
 
+// Delegate offset datetimes in terms of UTC primitive datetimes; this stores everything in the DB as UTC
+#[cfg(all(feature = "time", feature = "postgres_backend"))]
```

Comment:
It seems that none of the feature flags in this module are useful, because this entire module isn't loaded if we don't have the `time` and `postgres_backend` features enabled

Reply:
Maybe we can clean them up in a separate PR? I'm trying to follow existing conventions here.

Reply:
Ah so it seems they are actually useful :)